### ISSUE TITLE
Gross CO2 removals missing in emission variables

### ISSUE THREAD

**Issue by flohump** [IAMconsortium/common-definitions#224]
There are variables for gross CO2 emission and carbon dioxide removal but gross CO2 removals are currently missing in emission variables. I discussed with @merfort that it would be of high value to add gross CO2 removals including sub-categories espcially for AFOLU.
This would allow to analyze how gross CO2 emissions and gross CO2 removals add-up to net CO2 emissions. With the CDR variables this is not possible for two reasons:
- not all negative CO2 emissions are considered as CDR, e.g. regrowth of vegetation on abandoned land or regrowth of timber plantations for wood production
- the CDR variables are defined as positive 

@jkikstra @IAMconsortium/common-definitions-emissions @IAMconsortium/common-definitions-land 
What do you think about this proposal?

**Comment by strefler** [comment]
Agree! We prioritized net removals for the first step, but I think it would be valuable to disaggregate those into net removals and remaining emissions associated with CDR technologies. We should probably discuss the concrete structure, and also clear definitions to avoid confusion as to where emissions should be reported.

**Comment by danielhuppmann** [comment]
I noticed that we now have almost-duplicate variables

```
- Carbon Removal:
    description: Gross removals of carbon dioxide (CO2) from atmospheric origin
      or biomass through deliberate human activities
    unit: Mt CO2/yr
```
introduced via #69 

```
- Gross Removals|CO2:
    description: Gross removals of carbon dioxide (CO2) excluding positive emissions,
      including removals from bioenergy with CCS (BECCS), direct air carbon capture
      and agriculture, forestry and other land use (AFOLU)
```
introduced via #238

Looking at SoCDR Edition 3 scenarios, models report both but with different values (and sometimes different sign).

Any explanation?

**Comment by gidden** [comment]
As @danielhuppmann has pointed out, some teams are reporting `Gross Removals` as negative numbers (see image).

<img width="1888" height="1291" alt="Image" src="https://github.com/user-attachments/assets/eff9646b-618f-4a7f-97ab-71d12819ae91" />

As a fix, I propose that we include a check on scenario submission. I think we can say that both variables should be positive (though if you talk to Thomas Gasser, he will say land-based CDR could be negative given its ambiguous definition as a carbon flux; in fact OSCAR provided some negative results in our alignment paper - I would be curious what other land use colleagues say). To be a bit more general, we can say that novel CDR should be strictly positive. Additionally, for any variable where there is a 1-1 definition mapping between net and gross, then gross should be strictly greater than or equal to net.

**Comment by danielhuppmann** [comment]
FYI, "Carbon Removal" and all sub-categories already have a lower bound which is checked for all submissions to projects using **common-defintiions**, see [here](https://github.com/IAMconsortium/common-definitions/blob/main/definitions/variable/emissions/carbon-removal.yaml#L1).

Non-negativity validation for "Gross Removals|CO2" is proposed to be added in #417.

But I still don't understand the (intended?) difference between the two variables.

**Comment by flohump** [comment]
@danielhuppmann @gidden — picking up on your posts: I don't think `Carbon Removal`
and `Gross Removals|CO2` are duplicates. `Carbon Removal` is deliberate CDR, whereas
`Gross Removals|CO2|AFOLU` is broader and also includes land regrowth — so different
values are expected. The part that genuinely needs fixing is the differing sign,
which I agree on.

I introduced the `Intentional`/`Unintentional` split under `Gross Removals|CO2|AFOLU`
in #238, and in hindsight it wasn't a good call. The two sub-categories have been
reported only patchily and inconsistently across the ScenarioMIP submissions, and
the distinction doesn't really hold up. The State of CDR 3rd edition (Edwards et al.
2026, p. 25) counts land regrowth — including on abandoned land — as conventional
CDR:

> "We include forest regrowth on abandoned agricultural land in our summary
> estimates of CDR, despite debate over whether it constitutes direct human
> intervention."

So I support dropping the two sub-categories `…|Intentional` and `…|Unintentional`
(relevant to #384, which currently relocates them rather than dropping them).
Also tagging @strefler here.

The aggregate `Gross Removals|CO2|AFOLU` is a different matter — the content it
captures is what we need to retain: it is reported by all teams in ScenarioMIP, and
it is currently the only variable that captures the regrowth on abandoned
agricultural land — exactly the flux the quote above counts as CDR, but which the
`Carbon Removal` variables do not yet include. So if `Gross Removals` is to be
retired, `Carbon Removal|Land Use` should first be broadened to cover this regrowth;
until then it's worth keeping.

Whatever we land on here, I'd ask that none of it reach the ScenarioMIP template
before submissions are final — the submission workflow (`iiasa/ssp-submission-workflow`)
currently picks up changes from `main` automatically, so it would first need to be
frozen to a fixed version.

**Comment by strefler** [comment]
If I remember correctly, the idea was that Carbon Removal should report gross CDR (so the definition is correct). Apparently this is how Carbon Removal was understood so far, so sticking with this would avoid confusion. However, at least for some options like AR and OAE it would also be valueable to report Net Removals as well. Both should be reported as positive numbers, this is the difference to emissions, where CDR variables should be negative.

**Comment by strefler** [comment]
I've started a new issue that should lead to a PR that fixes some inconsistencies (#421). I think we agreed that Carbon Removal should be gross removals, Gross CO2 Removals should be removed, and Net Carbon Removal should be added where necessary. Nevertheless, there are still some issues in the land CO2 that need fixing, e.g. removal of "Intentional" and "Unintentional", separation of fires, new land hierarchy, and also a clear set of Net Carbon Removal variables and definitions. We've discussed this already, could a land expert start another issue / PR that takes these things up please? @flohump (also Thomas Grasser was part of this, but I don't find him here).
